src/ucp/core/ucp_worker.c

```diff
@@ -1574,9 +1574,9 @@ static void ucp_worker_add_feature_rsc(ucp_context_h context,
     ucs_string_buffer_appendf(strb, ") ");
 }
 
-void ucp_worker_print_used_tls(const ucp_ep_config_key_t *key,
-                               ucp_context_h context,
-                               ucp_worker_cfg_index_t config_idx)
+static void ucp_worker_print_used_tls(const ucp_ep_config_key_t *key,
```

Comment:
also change H file
(remove from H file)

Reply:
it was removed